cloud/amazon/ec2_vpc_vpn_connection.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This Ansible library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = """
+---
+module: ec2_vpc_vpn_connection
+short_description: create or remove a vpn connection
+description:
+    - create or remove a vpn connection
+version_added: "2.1"
+author: Kristian Jones (@klj613)
+options:
+  name:
+    description:
+      - Unique name of the vpn connection
+    required: true
+  type:
+    description:
+      - The type of the vpn connection
+    required: true
+  customer_gateway_id:
+    description:
+      - ID of the customer gateway id
+    required: true
+  vpn_gateway_id:
+    description:
+      - ID of the customer gateway id
+    required: true
+  static_routes_only:
+    description:
+      - Use static routes only instead of dynamic
+    required: false
+    default: false
+  wait:
+    description:
+      - Have ansible wait until the vpn connection is fully ready
+    required: false
+    default: false
+  wait_timeout:
+    description:
+      - How long to wait until the vpn connection is fully ready
```

Comment:
I assume this is seconds, so

`How long to wait (in seconds) until the VPN connection is fully ready.`